tests/Issues/PartialValueDocblockUpdate/Fixture/add_default_value_to_route.php.inc

```diff
@@ -1,42 +1,38 @@
 <?php
 
-use Shopware\Core\Framework\Context;
+namespace Rector\Core\Tests\Issues\PartialValueDocblockUpdate\Fixture;
+
 use Shopware\Core\Framework\Routing\Annotation\Since;
-use Symfony\Component\HttpFoundation\JsonResponse;
 use Symfony\Component\Routing\Annotation\Route;
 
-class Test {
+final class AddDefaultValueToRoute
+{
     /**
      * @Since("6.4.0.1")
      * @Route("/api/_admin/reset-excluded-search-term", name="api.admin.reset-excluded-search-term", methods={"POST"})
-     *
-     * @return JsonResponse
```

Comment:
No, this `@return` here is to reproduce the error, and  the order need to stay

Reply:
see https://github.com/rectorphp/rector-src/pull/1862 for why it is needed for reproduce the error

Reply:
I don't see any exact reason for the `@return` to be there. It's only in original fixture.

This should test the Doctrine annotation is changed propperly

Reply:
Yes, but without the `@return JsonResponse`, the original issue was never exists, it was issue with SpacelessPhpDocTagNode next transformation

Reply:
I see. I'll add it back then